content/doc/advanced/customizing-gitea.en-us.md

```diff
@@ -17,6 +17,22 @@ menu:
 
 The main way to customize Gitea is by using the `custom` folder. This is the central place to override and configure features.
 
+## Specify my customerize folder
+
+Set the env `GITEA_CUSTOM`.
+
 ## Customizing /robots.txt
 
 To make Gitea serve your own `/robots.txt` (by default, an empty 404 status is served), simply create a file called `robots.txt` in the `custom` folder with the [expected contents](http://www.robotstxt.org/). **Note** that you have to restart Gitea for it to notice the new `robots.txt`.
+
+## Customizing licenses/readme/locales/labels/gitignores
+
+Put your customerize files into `custom/options/license`/`custom/options/readme/`/`custom/options/locale`/`custom/options/label/`/`custom/options/gitignore`
```

Comment:
As you introduced `GITEA_CUSTOM` above, maybe this document could be consistent and use `${GITEA_CUSTOM}` instead of just `custom` ?